releasetasks/templates/firefox/routes.yml.tmpl

```diff
@@ -0,0 +1,23 @@
+# Routing Templates: releases v2!
```

Comment:
In bug 1286075, gecko.v2 routes are now in python (taskcluster/taskgraph/transforms/task.py) rather than in yml / json. Would it make sense to similarly have the release routes in a python file? (Either task.py or some equivalent in the release world).